back to home button

kayliezhao opened the pull request:
purple-300 shows gray on my screen
![image](https://github.com/acm-ucr/witl-website/assets/151389729/749b6f00-8f35-4496-9618-6bd46287da6f)

kayliezhao committed:
purple-300 shows gray on my screen

kayliezhao committed:
Merge branch 'dev' of https://github.com/acm-ucr/witl-website into kaylie/homeButton

kayliezhao committed:
I added the back to home button for the 404 not found page

kayliezhao commented:
updated code + button on local host screenshot 
<img width="724" alt="Screenshot 2024-06-28 115437" src="https://github.com/acm-ucr/witl-website/assets/151389729/0d2d7def-0d9a-41d8-acc8-9a76fae3a8fd">

stanleylew5 committed:
Merge branch 'dev' into kaylie/homeButton

stanleylew5 commented:
- ill have you change the background color of the 404 page to match the purple in the figma
- pull from dev and make sure your code is inside of the not found component (this is the jsx file, not the js file)
- and make the back to home font matches the one in the figma
- round the corners of your div container too

feel free to ask for help if anything I said was confusing

kayliezhao committed:
Merge branch 'dev' of https://github.com/acm-ucr/witl-website into kaylie/homeButton

kayliezhao committed:
Merge branch 'dev' of https://github.com/acm-ucr/witl-website into kaylie/homeButton

kayliezhao committed:
changed home button font and added background color

kayliezhao commented:
Changed background color to match the figma, added purple-400 for color hex of bg color, fixed font

<img width="405" alt="Screenshot 2024-06-29 110220" src="https://github.com/acm-ucr/witl-website/assets/151389729/653c214e-9d54-4618-af25-a4ede2fb8a18">
<img width="684" alt="Screenshot 2024-06-29 110156" src="https://github.com/acm-ucr/witl-website/assets/151389729/b74abfe7-f8c5-4c58-8244-db8cdedf3079">
<img width="451" alt="Screenshot 2024-06-29 110128" src="https://github.com/acm-ucr/witl-website/assets/151389729/6a18cb06-30d3-4f79-8406-fae32463a004">

kayliezhao committed:
Merge branch 'dev' into kaylie/homeButton

kayliezhao committed:
added to notfound jsx, changed font & bg

kayliezhao committed:
Merge branch 'kaylie/homeButton' of https://github.com/acm-ucr/witl-website into kaylie/homeButton
